plugins/performance-lab/includes/admin/load.php

```diff
@@ -228,8 +225,14 @@ function perflab_enqueue_features_page_scripts(): void {
 	wp_enqueue_style( 'thickbox' );
 	wp_enqueue_script( 'plugin-install' );
 
-	// Enqueue the a11y script.
-	wp_enqueue_script( 'wp-a11y' );
+	// Enqueue plugin activate AJAX script and localize script data.
+	wp_enqueue_script(
+		'perflab-plugin-activate-ajax',
+		plugins_url( 'includes/admin/plugin-activate-ajax.js', PERFLAB_MAIN_FILE ),
```

Comment:
I think we should use `plugin_dir_url()`, as that's what we use elsewhere for this purpose. I think `plugins_url()` is a lower-level function and is in fact used in `plugin_dir_url()`.

If we change to use `plugins_url()`, this should be first discussed in #1647, instead of arbitrarily changing here.
```suggestion
		plugin_dir_url( PERFLAB_MAIN_FILE ) . 'includes/admin/plugin-activate-ajax.js',
```